client/landing/stepper/declarative-flow/internals/steps-repository/blogger-intent/style.scss

```diff
@@ -104,14 +104,14 @@
 			&.disabled:focus,
 			&.disabled:hover,
 			&:disabled {
-				background: var(--studio-blue-20);
+				background: var(--color-accent-20);
 				cursor: default;
 			}
 			&:focus {
 				box-shadow: none;
-				outline: 2px solid var(--studio-blue-60);
+				outline: 2px solid var(--color-accent-60);
 				outline-offset: 1px;
-				background: var(--studio-blue-60);
+				background: var(--color-accent-60);
 			}
 			&:hover {
 				background-color: var(--wp-admin-theme-color-darker-10);
```

Comment:
Will want a --color-accent-60 here too, (we just replaced it with blue 60 a few minutes ago)

Reply:
Updated